plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonq/webview/Browser.kt

```diff
@@ -127,7 +128,7 @@ class Browser(parent: Disposable, private val webUri: URI, val project: Project)
         // language=HTML
         val jsScripts = """
             <script type="text/javascript" charset="UTF-8" src="$connectorAdapterPath"></script>
-            <script type="text/javascript" charset="UTF-8" src="$webUri" defer onload="init()"></script>
+            <script type="text/javascript" charset="UTF-8" src="${ assetRequestHandler.createResource("amazon-q.js", mynahAsset.inputStream()) }" defer onload="init()"></script>
```

Comment:
nit: To keep it consistent:
```suggestion
            <script type="text/javascript" charset="UTF-8" src="${ assetRequestHandler.createResource("amazonq-ui.js", mynahAsset.inputStream()) }" defer onload="init()"></script>
```